### PR TITLE
3rdParty Transcription Fixes

### PR DESCRIPTION
**Pull request by Qubad786**

### CODE REVIEW

**Comment by Qubad786** [comment]
@muzaffaryousaf this is ready.

File: VEDA_OS01/tests/test_transcripts.py [hunk 1]
```diff
@@ -1203,6 +1211,10 @@ def test_translations_retrieval(self):
             decode_func=json.loads,
         )
 
+        # Asserts edx-video-pipeline's video status
+        video = Video.objects.get(studio_id=self.video.studio_id)
+        self.assertEqual(video.video_trans_status, transcripts.VideoStatus.TRANSCRIPTION_READY)
```
Comment (muzaffaryousaf) [inline VEDA_OS01/tests/test_transcripts.py]: Nothing to check for IN_PROGRESS ?

Comment (Qubad786) [reply VEDA_OS01/tests/test_transcripts.py]: We don't have test coverage for pipeline delivery phase.

**Comment by Qubad786** [comment]
@yro I've added 3Play Media Translations retrieval to the Heal process for scheduling purpose. May you please review?

**Comment by yro** [comment]
👍 Looks good from my end.

**Comment by Qubad786** [comment]
@muzaffaryousaf I've rebase this as well.

**Pull request merged**